recipes/spirv-tools/all/conanfile.py

```diff
@@ -0,0 +1,123 @@
+        cmake.definitions["SPIRV-Headers_SOURCE_DIR"] = self.deps_cpp_info["spirv-headers"].rootpath
+
+        # There are some switch( ) statements that are causing errors
+        # need to turn this off
+        cmake.definitions["SPIRV_WERROR"] = False
+
+        cmake.definitions["SKIP_SPIRV_TOOLS_INSTALL"] = False
+        cmake.definitions["SPIRV_LOG_DEBUG"] = False
+        cmake.definitions["SPIRV_SKIP_TESTS"] = True
+        cmake.definitions["SPIRV_CHECK_CONTEXT"] = False
+        cmake.definitions["SPIRV_BUILD_FUZZER"] = False
+
+        cmake.configure(build_folder=self._build_subfolder)
+        self._cmake = cmake
+        return self._cmake
+
+    def build(self):
+        self._patch_sources()
+        cmake = self._configure_cmake()
+        cmake.build()
+
+    def _patch_sources(self):
+        # CMAKE_POSITION_INDEPENDENT_CODE was set ON for the entire
+        # project in the lists file.
+        tools.replace_in_file(os.path.join(self._source_subfolder, "CMakeLists.txt"),
+                              "set(CMAKE_POSITION_INDEPENDENT_CODE ON)", "")
+
+    def package(self):
+        self.copy(pattern="LICENSE*", dst="licenses", src=self._source_subfolder)
+        cmake = self._configure_cmake()
+        cmake.install()
+
+        tools.rmdir(os.path.join(self.package_folder, "lib", "pkgconfig"))
+        tools.rmdir(os.path.join(self.package_folder, "lib", "cmake"))
+
+        # SPIRV-Tools-shared is meant to be a shared-only c-only API for the library .
+        # it is built by the original CMakeLists.txt file. It is the same
+        # as the other libraries except only the C interface is exposed
+        # This library is being removed because it is only causing confusion.
+        # As a result, you are restricted to to using the C++ api only.
+        for bin_file in glob.glob(os.path.join(self.package_folder, "bin", "*SPIRV-Tools-shared.dll")):
+            os.remove(bin_file)
+        for lib_file in glob.glob(os.path.join(self.package_folder, "lib", "*SPIRV-Tools-shared*")):
+            os.remove(lib_file)
+
+    def package_info(self):
+        # TODO: set targets names when components available in conan
+        self.cpp_info.names["cmake_find_package"] = "SPIRV-Tools"
+        self.cpp_info.names["cmake_find_package_multi"] = "SPIRV-Tools"
+
+        # The spirv-tools CMAKE builds a SPIRV-Tools-shared.so which is
+        # apparantly only exports the C-interface of the library.
+        # The test_package.c is used when testing the c_only option
+        # The C-interface is ALWAYS built as a shared-object as per the original
+        # CMakeLists.txt file
+        self.cpp_info.libs.append("SPIRV-Tools-reduce")
+        self.cpp_info.libs.append("SPIRV-Tools-link")
+        self.cpp_info.libs.append("SPIRV-Tools-opt")
+        self.cpp_info.libs.append("SPIRV-Tools")
+
+        if self.settings.os == "Linux":
+            self.cpp_info.system_libs.append("rt") # for SPIRV-Tools
```

Comment:
```suggestion
            self.cpp_info.system_libs.extend(["m", "rt"]) # m for SPIRV-Tools-opt, rt for SPIRV-Tools
```

`objdump -o libSPIRV-Tools-opt.so`:

```
...
  NEEDED               libSPIRV-Tools.so
  NEEDED               libstdc++.so.6
  NEEDED               libm.so.6
  NEEDED               libc.so.6
  SONAME               libSPIRV-Tools-opt.so
...
```